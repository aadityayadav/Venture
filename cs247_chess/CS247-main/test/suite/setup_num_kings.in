setup
done
+ K e1
done
+ k e8
+ k e7
done
- e7
done
quit
