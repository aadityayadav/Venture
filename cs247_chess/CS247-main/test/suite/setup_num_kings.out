The board must contain exactly one king for each colour!
The board must contain exactly one king for each colour!
The board must contain exactly one king for each colour!
____k___
________
________
________
________
________
________
____K___
